tests/test_optimizely.py

```diff
@@ -5139,6 +5168,49 @@ def test_sdk_settings__accept_cache_size_and_cache_timeout(self):
         mock_logger.error.assert_not_called()
         client.close()
 
+    def test_sdk_settings__accept_default_when_cache_timeout_zero(self):
+        mock_logger = mock.Mock()
+        sdk_settings = OptimizelySdkSettings(segments_cache_timeout_in_secs=0)
+        client = optimizely.Optimizely(
+            json.dumps(self.config_dict_with_audience_segments),
+            logger=mock_logger,
+            settings=sdk_settings
+        )
+        segments_cache = client.odp_manager.segment_manager.segments_cache
+        self.assertEqual(segments_cache.timeout, 0)
+
+        mock_logger.error.assert_not_called()
+        client.close()
+
+    def test_sdk_settings__accept_default_when_cache_size_zero(self):
```

Comment:
This test is a subset of the next one, maybe change it to test None becomes default?